src/patternfly/components/Accordion/accordion-expanded-content.hbs

```diff
@@ -0,0 +1,8 @@
+<dd class="pf-c-accordion__expanded-content{{#if accordion-expanded-content--IsExpanded}} pf-m-expanded{{/if}}{{#if accordion-expanded-content--IsFixed}} pf-m-fixed{{/if}}{{#if accordion-expanded-content--modifier}} {{accordion-expanded-content--modifier}}{{/if}}"
+  {{#unless accordion-expanded-content--IsExpanded}}hidden{{/unless}}
+  {{#if accordion-expanded-content--attribute}}
+    {{{accordion-expanded-content--attribute}}}
+  {{/if}}>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis molestie lorem lacinia dolor aliquet faucibus. Suspendisse gravida imperdiet accumsan. Aenean auctor lorem justo, vitae tincidunt enim blandit vel. Aenean quis tempus dolor. Lorem ipsum dolor sit amet, consectetur adipiscing elit Bla Bla Blee Bla. 
```

Comment:
can you move this out of here and into the examples. That way you can add extra content for the fixed example to show how it scrolls.